common-template/src/main/java/com/redhat/cloud/notifications/qute/templates/extensions/TimeAgoFormatter.java

```diff
@@ -0,0 +1,81 @@
+package com.redhat.cloud.notifications.qute.templates.extensions;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.function.Function;
+
+// Based on: https://github.com/RedHatInsights/frontend-components/blob/86b02e3c206663d640d30b32087061c04db1595d/packages/components/src/DateFormat/helper.js#L4-L19
+public class TimeAgoFormatter {
```

Comment:
Could we move this class instead of duplicating it?

Reply:
sure